utils/Felt.go

```diff
@@ -87,3 +90,43 @@ func FeltArrToBigIntArr(f []*felt.Felt) []*big.Int {
 	}
 	return bigArr
 }
+
+// StringToByteArrFelt converts string to array of Felt objects.
+//
+// Parameters:
+// - s: string/bytearray to convert
+// Returns:
+// - []*felt.Felt: the array of felt.Felt objects
+// - error: an error, if any
+func StringToByteArrFelt(s string) ([]*felt.Felt, error) {
```

Comment:
What is the motivation for this function? Currently it's not being used so it's unclear why it should be in the codebase

Reply:
This utility method is useful for interacting with contracts that accept `ByteArray` as an input parameter, similar to the `HexToFelt` `HexArrToFelt` helper method. The `FunctionCall` can only take an array of felts as its payload, and this function assists in the serialization process, eliminating the need to find and go through the documentation.